instructure/canvas-android
# [MBL-17610][Parent] Implement help dialog

Test plan:
- Select Help option from the Navigation Drawer.
- Try out all available options at the dialog.
- Compare the available options with the previous Parent app
- All three application should be tested because the Report a Problem dialog has been refactored

refs: MBL-17610
affects: Student, Teacher, Parent
release note: none

## Checklist
